### PR TITLE
Update CRAN comments and version for next release

### PR DESCRIPTION
**Pull request by karawoo**
In prep for the next release of `scales` I've updated the version number and cran-comments.md. Is this the right information to include? This will be my first CRAN submission 😮 so I'm not sure what info they need.

Also I might've misinterpreted the instructions in our doc—I only built on Windows with R-devel, not R-release. Let me know if you want me to re-do that on both.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Package: scales
-Version: 0.4.1.9002
+Version: 0.5.1
```
Comment (hadley) [inline DESCRIPTION]: This should be 0.5.0

File: cran-comments.md [hunk 2]
```diff
@@ -1,22 +1,53 @@
...
-0 errors | 0 warnings | 1 notes
-
-* License components with restrictions and base license permitting such:
-  MIT + file LICENSE
+0 errors | 0 warnings | 0 notes
 
 ## Reverse dependencies
 
-This was minor release that only added new features, but I ran revdep checks on the 160 reverse dependencies just to be sure (complete results at  https://github.com/hadley/scales/blob/master/revdep).  I saw no problems related to scales.
+I ran revdep checks on the 272 reverse dependencies (complete results at  https://github.com/hadley/scales/blob/master/revdep). There were issues with three packages; one of these issues appears unrelated to `scales`.
```
Comment (hadley) [inline cran-comments.md]: Did you inform the maintainers?

Comment (karawoo) [inline cran-comments.md]: Yes, for DiagrammeR and incidence.

Comment (hadley) [inline cran-comments.md]: In that case, I think you're probably providing a bit too much detail to CRAN. You can just say "the maintainers were informed of these issues on xxx xxxxx xxx". CRAN mainly wants to know that you've done due diligence.

**Pull request merged**